sgx-types/src/secs.rs

```diff
@@ -66,8 +66,16 @@ testaso! {
 }
 
 impl Secs {
-    /// TODO: The max size of an enclave should come from CPUID, which is inaccessible from within SGX.
-    pub const SIZE_MAX: u64 = 0x1_000_000_000;
+    /// Returns the maximum enclave size for 64bit in bytes.
+    /// CPUID.(EAX=12H, ECX=0H) enumerates Intel SGX capability;
+    /// For more on CPUID enumeration leaves, see 37.7.2 and Table 37-4.
+    pub fn max_size_64bit() -> Option<u64> {
```

Comment:
Name this `max_size()`. We don't support 32bit enclaves.

Reply:
Fixed.